### ISSUE TITLE
Unmet Dependencies (Elementary OS Luna)

### ISSUE THREAD

**Issue by ykbks** [sarim/ibus-avro#86]
Hello there! Thank you so much for this awesome piece of software. It has changed the way we use Bangla for the better.

I have a small problem though, I cannot seem to be install it on my Elementary OS Luna (Ubuntu 14.10 based). Here's what it says:

![Image] (http://i.imgur.com/dhZ1zeZ.png)

Any idea how to resolve this?
Thank you in advance!

**Comment by sarim** [comment]
try `sudo apt-get install ibus-avro-quantal`

**Comment by ykbks** [comment]
Thank you for your reply.
Sadly, I end up on the same place. It seems the installation is trying to
download some components that already exist in a higher version than what
it wants to download.

**Comment by sarim** [comment]
What is the exact version of your elementary os? from wiki it says Elementary os luna is based on ubuntu 12.10, not 14.10

**Comment by ykbks** [comment]
Yes. My bad. I have tried a few things and it seems other than the look and a few apps, I'm actually stuck with a very old version of Ubuntu. I am changing the OS for good.
Thank you again for the support.

**Comment by apandada1** [comment]
Luna is based on 12.04 LTS. So the original ibus-avro package is to be
installed

On Mon, Mar 23, 2015 at 7:42 PM, Yousuf Kamal <notifications@github.com>
wrote:

> Yes. My bad. I have tried a few things and it seems other than the look
> and a few apps, I'm actually stuck with a very old version of Ubuntu. I am
> changing the OS for good.
> Thank you again for the support.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sarim/ibus-avro/issues/86#issuecomment-85015199>.
>



-- 
*Archisman Panigrahi. *

*I am learning Software Developing.*

*My Code: *https://github.com/apandada1/

*Supporting Greener Computing* - Do you really need to print this email?
